apps/plugin-browser/src/shared/badge.ts

```diff
@@ -0,0 +1,21 @@
+import browser from "webextension-polyfill";
+
+export const clearBadge = () => {
+  void browser.action.setBadgeText({ text: "" });
+};
+
+export const setLoadingBadge = () => {
+  void browser.action.setBadgeText({ text: "💭" });
+  void browser.action.setBadgeBackgroundColor({ color: "#006DC3" });
+};
+
+export const setErroredBadge = () => {
+  void browser.action.setBadgeText({ text: "❌" });
+  void browser.action.setBadgeBackgroundColor({ color: "black" });
+};
+
+export const setSuccessBadge = (num: number) => {
+  void browser.action.setBadgeBackgroundColor({ color: "#78B040" });
```

Comment:
```suggestion
  void browser.action.setBadgeBackgroundColor({ color: customColors.lime[70] });
```

Reply:
Thank you, fixed in https://github.com/hashintel/hash/pull/3360/commits/f0a263efca4552a51e0c6924f635187caaf961e7